features/cache.feature

```diff
@@ -133,6 +133,56 @@ Feature: Managed the WordPress object cache
       Error: Could not replace object 'bar' in group 'foo'. Does it not exist?
       """
 
+    When I run `wp cache flush-group add_multiple`
+    Then STDOUT should be:
+      """
+      Success: Cache group 'add_multiple' was flushed.
+      """
```

Comment:
You'll want to move this step to a new scenario so you can control it with `@require-wp-6.1`